src/settings.ts

```diff
@@ -758,3 +758,11 @@ export const FIREBASE_STORAGE_IMAGES_ORIGIN =
   "https://firebasestorage.googleapis.com/v0/b/sparkle-burn.appspot.com/o/";
 export const FIREBASE_STORAGE_IMAGES_IMGIX_URL =
   "https://sparkle-burn-users.imgix.net/";
+
+// Helper values that can be safely used in places that might re-render but don't have useMemo/useCallback
+export const ALWAYS_EMPTY_OBJECT = {};
+Object.freeze(ALWAYS_EMPTY_OBJECT);
+export const ALWAYS_EMPTY_ARRAY = [];
+Object.freeze(ALWAYS_EMPTY_ARRAY);
+export const ALWAYS_NOOP_FUNCTION = () => {};
+Object.freeze(ALWAYS_NOOP_FUNCTION);
```

Comment:
```suggestion
export const ALWAYS_EMPTY_OBJECT = Object.freeze({});
export const ALWAYS_EMPTY_ARRAY = Object.freeze([]);
export const ALWAYS_NOOP_FUNCTION = Object.freeze(() => {});
```

Reply:
This was my original code.

> :) approving, not sure though if we'll be able to use these consts everywhere because of the typings.

☝️ 
That was the reason for the separate lines of the `Object.freeze()`.
Let TS still think they're not ReadOnly<>.

Hence, not accepting the suggestion.